app/views/forms/show.html.erb

```diff
@@ -1,22 +1,22 @@
 <% set_page_title(@form.name) %>
 
 <% content_for :back_link do %>
-  <%= link_to "Back", root_path, class: "govuk-back-link" %>
+  <%= link_to t("forms.form_overview.back"), root_path, class: "govuk-back-link" %>
 <% end %>
```

Comment:
Could we not use`<% content_for :back_link, govuk_back_link_to(root_path) %>` like the other pages?

Reply:
Good spot, have updated!